pkgs/os-specific/linux/tuxedo-keyboard/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, kernel, kmod }:
+
+stdenv.mkDerivation rec {
+  name = "tuxedo-keyboard-${kernel.version}-${version}";
+  version = "2019-08-26";
+
+  src = fetchFromGitHub {
+    owner = "tuxedocomputers";
+    repo = "tuxedo-keyboard";
+    rev = "d65e76e84cfd8169591fc2a0a7c9219fa19da1b5";
+    sha256 = "1s48qpwybwh5pwqas2d1v2a7x4r97sm4hr9i4902r1d7h384bv17";
+  };
+
+  preConfigure = ''
+    substituteInPlace Makefile --replace '/lib/modules/' "${kernel.dev}/lib/modules/"
+    substituteInPlace Makefile --replace '$(shell uname -r)' "${kernel.modDirVersion}"
```

Comment:
Can't this be replaced with `makeFlags = [ "KDIR=${kernel.dev}/lib/modules/${kernel.modDirVersion}/build" ];` ?

Reply:
That works like a charm! Thanks!